packages/@storefront/flux-capacitor/src/core/reducers/data/past-purchases.ts

```diff
@@ -43,17 +49,17 @@ export const DEFAULTS: State = <any>{
   query: '',
   sort: {
     items: [{
-      field: 'Default',
+      field: SORT_FIELDS[SORT_ENUMS.DEFAULT],
       descending: true,
       type: SORT_ENUMS.DEFAULT,
```

Comment:
Don't need both `type` and `field` as unique identifiers, just use `field`.

Reply:
Noted. Will drop `type` and associated enum values.

Reply:
Addressed, resolving.